concent_api/core/views.py

```diff
@@ -178,6 +181,12 @@ def receive(request, _message):
             last_undelivered_message_status
         )
 
+    if isinstance(decoded_message_data, message.concents.ForcePaymentCommitted):
```

Comment:
There is no request to `receive` in this use case. `ForcePaymentCommitted` for provider should be returned in the same request which is sent to `send`.

Reply:
I didn't delete it after our conversation about sending and receiving messages. However now it's fixed